### ISSUE TITLE
FinInG package broken by master branch

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap#498]
With FinInG package loaded, the following happens in master branch but not in stable-4.8:
```
gap> SplitCayleyHexagon(3);
Error, no method found! For debugging hints type ?Recovery from NoMethodFound
Error, no 1st choice method found for `TriangulizeMat' on 1 arguments called from
TriangulizeMat( x 
); at /Users/alexk/GITREPS/gap/pkg/fining/lib/polarspace.gi:1701 called from
VectorSpaceToElement( geom, Unpack( v ) 
) at /Users/alexk/GITREPS/gap/pkg/fining/lib/polarspace.gi:1674 called from
VectorSpaceToElement( ps2, ShallowCopy( x!.obj ) * change 
) at /Users/alexk/GITREPS/gap/pkg/fining/lib/morphisms.gi:639 called from
em!.fun( t 
) at /Users/alexk/GITREPS/gap/pkg/fining/lib/morphisms.gi:185 called from
func( C[i] ) at /Users/alexk/GITREPS/gap/lib/coll.gi:745 called from
...  at line 2 of *stdin*
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk> x;
[ 0*Z(3), 0*Z(3), 0*Z(3), 0*Z(3), 0*Z(3), Z(3)^0, 0*Z(3) ]
brk> 
```
@stevelinton bisected this to the change that makes IsPlistRep imply HasLength, namely https://github.com/gap-system/gap/commit/7f00bbb69ea2902fbb1bead8e75df4971366ee79

**Comment by fingolfin** [comment]
I discussed this with Jan De Beule. We discovered that the problem was that (a) the rank of `IsPlistRep` went up from 2 to 21, and (b) this caused a method for FinInG `VectorSpaceToElement` to be called which didn't expect to be called...

But Jan will take care of this in FinInG. I see no other packages in the distro using it, so I hope this is the only case of this kind... we'll see...

**Comment by alex-konovalov** [comment]
Fixed in the new version of FinInG.

**Comment by fingolfin** [comment]
When you say "fixed in new version", which version is that? Because the official fining page lists 1.0, which clearly still has the bug, and I just obtained a version '1.3", which also has the bug...

**Comment by fingolfin** [comment]
This is fixed in fining 1.3.3, which will hopefully soon ship with GAP 4.8.2
